README.docker.md

```diff
@@ -0,0 +1,32 @@
+# Staytus
+
+Staytus is a complete solution for publishing the latest information about
+any issues with your web applications, networks or services. Along with
+absolutely beautiful public & admin interfaces, Staytus is a powerful tool for
+any organization with customers that rely on them to be online 24/7.
+
+* [Check out the live demo](http://demo.staytus.co)
+* [Read the roadmap](https://github.com/adamcooke/staytus/blob/master/ROADMAP.md)
+* [Report a bug](https://github.com/adamcooke/staytus/issues/new?labels=bug)
+* [Ask a question](https://github.com/adamcooke/staytus/issues/new?labels=question)
+
+![Screenshot](https://s.adamcooke.io/15/iOzvtk.png)
+
+## Setup with Docker
+
+1. [Install Docker locally](https://docs.docker.com/installation/)
+2. [Install docker-compose locally](https://docs.docker.com/installation/)
+3. Now run `docker-compose -d up`, to start Staytus with the database container (You have to be in the directory of your Staytus `docker-compose.yml` for this).
+4. Go to [http://localhost:80](http://localhost:80) and follow the instructions to configure Staytus
```

Comment:
I changed the run command to `curl https://github.com/adamcooke/staytus/blob/master/docker-compose.yml | docker-compose -f - up -d`, as someone suggested in the comment https://github.com/docker/compose/issues/1818#issuecomment-164807629.